test-fixtures/ci/get-next-pr-version

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
```

Comment:
I think we should move this line afte the required header, in L6, if possible

Reply:
You mean move this under the license info?

I think the shebang is required to be on the first line otherwise it won't be recognized as the instruction for which interpreter to use.

I can remove it though since I'm calling the script with `bash` if you'd like the license info to be at the top.

**The below context isn't strictly required reading**:

With that said, it should only matter if the script is called like this `./test-fixtures/ci/get-next-pr-version`

since I'm calling the script with `bash test-fixtures/ci/get-next-pr-version` it doesn't really matter and is more of a safeguard when someone else uses it.

However, double-exception, I noticed I refactored out the use of `mkfifo`  and the use of associative arrays in favor of the simpler indexed arrays and `mktemp` file approach and no longer use anything required in bash 4.0. But bash 3.0 would still be required.

Removing the `shebang` from the first line and running the script using just `sh` causes the script to fail...but, some systems link `sh` to `bash` anyway so you don't actually end up using `shell`, which is the case for my mac, and I have to _force_ the script to use a 'pure' `sh` command with no linking to bash.

All that to say, the risk is extremely small that there's actually going to be an issue anywhere by `removing the shebang` altogether, which I can do.

That being the case, I think any shell version would work as it's rare for a system to not link `sh` to `bash` and for a `ci` system to not use the `latest version of bash`.